tests/settings/test_settings.py

```diff
@@ -18,3 +18,28 @@ def test_invalid_storage() -> None:
         assert False, f"'validate_settings' raised an exception {exc}"
     finally:
         cluster[0]["storage_sets"].remove("non_existing_storage")
+
+
+def test_topics_sync() -> None:
+    from copy import deepcopy
+
+    from snuba.settings.validation import validate_settings
+    from snuba.utils.streams.topics import Topic
+
+    all_settings = {
+        key: value
+        for key, value in settings.__dict__.items()
+        if not key.startswith("__") and not callable(key)
+    }
+    topic_map = all_settings["KAFKA_TOPIC_MAP"]
+    default_map = deepcopy(topic_map)
```

Comment:
I think you may not need this deepcopy as you are not updating the values of this mapping.

Reply:
I realized that there was a mistake with this test that I wrote -- it does not accurately update KAFKA_TOPIC_MAP as I wanted. 

all_settings[KAFKA_TOPIC_MAP] should change so that it imports and includes all of the Topics enum values, and then gets validated by validate_settings(). After validate_settings() it should go back to its "default" state, which is why a deepcopy is needed if I understand correctly.